rama-http-types/src/body_ext.rs

```diff
@@ -80,6 +132,13 @@ impl<B: Into<crate::Body> + Send + 'static> BodyExtractExt for B {
         let bytes = body.to_bytes();
         String::from_utf8(bytes.to_vec()).context("parse body as utf-8 string")
     }
+
+    fn into_data_stream(
+        self,
+    ) -> impl Stream<Item = Result<Self::StreamData, Self::StreamError>> + Send {
+        let body: crate::Body = self.into();
```

Comment:
```suggestion
        let body = self.into();
```

Don't think you need this, right? Given that you specify it as `B: Into<crate::Body>`, or do you need it?

Reply:
Oh yeah, you're right.